registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflowmanager/verticle/WorkflowInternalActionVerticle.java

```diff
@@ -232,10 +259,34 @@ private void processCompleteAsFailed(WorkflowInternalActionDTO workflowInternalA
 		registrationStatusDto.setLatestTransactionTypeCode(RegistrationTransactionTypeCode.INTERNAL_WORKFLOW_ACTION.toString());
 		registrationStatusDto.setSubStatusCode(StatusUtil.WORKFLOW_INTERNAL_ACTION_SUCCESS.getCode());
 		registrationStatusService.updateRegistrationStatus(registrationStatusDto, MODULE_ID, MODULE_NAME);
-		sendWorkflowCompletedWebSubEvent(registrationStatusDto);
+		if (subWorkflowDtos != null && !subWorkflowDtos.isEmpty()) {
+			Map<String, String> tags = new HashMap<String, String>();
+			tags.put(workflowInternalActionDTO.getReg_type() + "_FLOW_STATUS",
+					RegistrationStatusCode.FAILED.toString());
+			packetManagerService.addOrUpdateTags(workflowInternalActionDTO.getRid(),
+					tags);
+			SubWorkflowDto subWorkflowDto = subWorkflowDtos.get(0);
+			InternalRegistrationStatusDto mainFlowregistrationStatusDto = registrationStatusService
+					.getRegistrationStatus(
+					subWorkflowDto.getRegId(), subWorkflowDto.getParentProcess(), subWorkflowDto.getParentIteration());

```

Comment:
pass null for now for the 4th argument everywhere for getRegistrationStatus method is called with subWorkflow details

Reply:
fixed